docs/generated/devkit/index.md

```diff
@@ -927,10 +927,11 @@ stored in the daemon process. To reset both run: `nx reset`.
 
 #### Parameters
 
-| Name               | Type      |
-| :----------------- | :-------- |
-| `opts`             | `Object`  |
-| `opts.exitOnError` | `boolean` |
+| Name                      | Type      |
+| :------------------------ | :-------- |
+| `opts`                    | `Object`  |
+| `opts.exitOnError`        | `boolean` |
+| `opts.resetDaemonClient?` | `boolean` |
```

Comment:
this looks to be from changes in 8d1f284048b4b1cb1d7080d5a97b9719137150e9
just went ahead and included it 🤷